synapse/handlers/room_member.py

```diff
@@ -154,6 +154,30 @@ def remote_join(self, remote_room_hosts, room_id, user, content):
         )
         yield user_joined_room(self.distributor, user, room_id)
 
+    @defer.inlineCallbacks
+    def _remote_reject_invite(self, remote_room_hosts, room_id, target):
```

Comment:
please could you add a docstring documenting the params and return value? I don't want to have to go spelunking to figure it out